crates/biome_rule_options/src/no_increment_decrement.rs

```diff
@@ -0,0 +1,9 @@
+use biome_deserialize_macros::Deserializable;
+use serde::{Deserialize, Serialize};
+#[derive(Default, Clone, Debug, Deserialize, Deserializable, Eq, PartialEq, Serialize)]
+#[cfg_attr(feature = "schema", derive(schemars::JsonSchema))]
+#[serde(rename_all = "camelCase", deny_unknown_fields, default)]
+pub struct NoIncrementDecrementOptions {
+    /// Allows unary operators ++ and -- in the afterthought (final expression) of a for loop.
+    pub allow_for_loop_afterthoughts: bool,
```

Comment:
We generally don't add options for new rules, because we want to have concrete use cases and/or actual user demand.

What's the source rule's default behavior for this? I would imagine it would be `true`. In any case we should adhere to the default behavior of the eslint rule.